### ISSUE TITLE
Move lists across boards?

### ISSUE THREAD

**Issue by veddox** [apankrat/nullboard#11]
Hey there! Good job on this app, I really like it! Pleasantly minimalistic, and good design choices on data storage/export.

One feature I would appreciate is the ability to move lists to another board. I started a board for a new project yesterday, but found out pretty soon that I need more space, so I had to expand to several boards. I didn't want to retype everything, but manually cutting/pasting JSON across exported files and then re-importing isn't particularly pleasant either.

In short: Moving lists across boards (if technically feasible) would greatly increase scalability.

**Comment by apankrat** [comment]
Let me see what I can do.

**Comment by gf-mse** [comment]
Not exactly that, but #63 shall allow one to _merge_ two boards, so all you would have to do is merge -- and then delete extra lists )
